lib/apiservers/engine/backends/container.go

```diff
@@ -58,6 +58,19 @@ import (
 
 const (
 	bridgeIfaceName = "bridge"
+
+	// MemoryMinMiB - the minimum allowable container memory size
+	MemoryMinMiB = 1024
+	// MemoryMaxMiB - the maximum allowable container memory size
+	MemoryMaxMiB = 16384
+	// MemoryDefaultMiB - the default container VM memory size
+	MemoryDefaultMiB = 4096
```

Comment:
the Container.ContainerVMSize field in lib/config has been around since inception for this, but yes, it's a scope question. That said, it may be faster to provide a mechanism to set defaults than to determine properly sane defaults that cannot easily be overridden :)